schedule/yast/maintenance/skip_registration.yaml

```diff
@@ -0,0 +1,12 @@
+---
+name: skip_registration
+vars:
+  YUI_REST_API: 1
+schedule:
+  registration:
+    - installation/registration/skip_registration
+  extension_module_selection: []
+  add_on_product:
+    - installation/module_selection/select_module_desktop
```

Comment:
this belongs to the key above.

Reply:
Nope, because in the default schedule there is a module in between that ruins the execution queue needed here

Reply:
oh that is messy! but sort of expected, could you please file a ticket to unify this?